.github/actions/rust-build-release/src/main.py

```
@@ -605,6 +614,55 @@ def _manifest_argument(manifest_path: Path) -> Path:
+            f"toolchain '{toolchain_name}' does not support target '{target_to_build}'"
+        )
+        raise typer.Exit(1)
+
+
+def _assemble_build_command(
+    decision: _CrossDecision,
+    target_to_build: str,
+    manifest_argument: Path,
+    features: str,
+    explicit_toolchain: str,
+    toolchain_name: str,
```

Ruff (0.15.12)

[warning] Unused function argument: `toolchain_name`

(ARG001)